test/integration/utilities_test.go

```diff
@@ -97,8 +98,8 @@ func RunCRCExpectFail(args ...string) (string, error) {
 }
 
 // Send command to CRC VM via SSH
-func SendCommandToVM(cmd string) (string, error) {
-	client := machine.NewClient(constants.InstanceName(), false,
+func SendCommandToVM(cmd string, preset crcPreset.Preset) (string, error) {
+	client := machine.NewClient(constants.InstanceName(preset), false,
```

Comment:
I see, so this `constants.InstanceName` is not preset dependent, and the changes we need in tests are all either directly to the `constants.InstanceName` or to the `SendCommandToVM` function which utilizes this. LGTM. Thanks for the changes!